.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,70 @@
+name: CI Build
+
+on:
+  - push
+
+jobs:
+  build:
+    name: Building with Scala ${{ matrix.scala }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        scala: [ 2.12.14, 2.13.6 ]
+
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java and Scala with JDK 11
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: 11
+
+      - name: Cache Local SBT Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.coursier/cache/v1
+            ~/.cache/coursier/v1
+          key: ${{ runner.os }}-sbt-cache-${{ hashFiles('**/*.sbt') }}-${{ hashFiles('project/build.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-sbt-cache-
+
+      - name: Check Formatting
+        run: sbt ++${{ matrix.scala }} scalafmtSbtCheck scalafmtCheckAll
+
+      - name: Run CI
+        run: ./scripts/ci.sh
+        env:
+          SCALA_VERSION: ${{ matrix.scala }}
+
+      - name: Upload Coverage Results to Coveralls
+        run: sbt ++${{ matrix.scala }} coveralls
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # This required because of the sonarcloud-github-action docker volume mapping.
+      - name: Prepare for Sonar Cloud Scan
+        run: |
+          find . -name scoverage.xml -exec sed -i \
+          's#/home/runner/work/import-export-udf-common-scala/import-export-udf-common-scala#/github/workspace#g' {} +
+
+      - name: Sonar Cloud Scan
+        uses: sonarsource/sonarcloud-github-action@master
```

Comment:
Does it run only once or with each version in matrix?

Reply:
I think it runs for both, added fix to only run for 2.13.